### ISSUE TITLE
Error: `Failed to generate the D2 diagram at`

### ISSUE THREAD

**Issue by nibblesnbits** [HiDeoo/astro-d2#14]
### Describe the bug

When I include any images in my d2 markup, such as

```d2
ElastiCache: {
  shape: image
  icon: /media/icons/elasticache.svg
}
```

I get the error `Failed to generate the D2 diagram at...`.  Removing the image reference fixes it, and I think it might be because d2 doesn't know to look into my `public` folder for the images.

### To Reproduce

Include any `icon` in a d2 diagram

### Expected behavior

The engine should be able to find the images

### How often does this bug happen?

Every time

### System Info

OS: Windows 11 23H2

**Comment by HiDeoo** [comment]
Thanks for your feedback :raised_hands:

This should now be fixed in version [`0.3.0`](https://github.com/HiDeoo/astro-d2/releases/tag/v0.3.0) that I just released. Note that this works like Astro’s [optimized asset support](https://docs.astro.build/en/guides/images/) and you will need to use a relative path to an image in your `src/` directory.

You can find a live example and documentation using relative paths in the [documentation](https://astro-d2.vercel.app/examples/icons/#relative-paths).

**Comment by nibblesnbits** [comment]
And a direct reference to using d2.  Thanks!  🎉🎉🎉
